src/clients/storage/CassandraTableStorageManager.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const asyncEachLimit = require('async/eachLimit');
+
+/** http://docs.datastax.com/en/developer/nodejs-driver/3.2/features/batch/
+ * TODO: ASYNC_BATCH_LIMIT will need to be tuned.
+ * It is best to keep the batch size small (in the order of 10s).
+ * If the batch size is greater than 5k, then the server will issue a warning.
+ */
+const ASYNC_BATCH_LIMIT = 10;
+
+/** Used to execute multiple INSERT, UPDATE, or DELETE statements
+ * @param {Client} client
+ * @param {Array<{mutation: string, params: Array}>} mutations
+ */
+function batchMutations(client, mutations, callback) {
+  asyncEachLimit(mutations, ASYNC_BATCH_LIMIT, (mutationBatch, asyncCB) => processMutation(client, mutationBatch, asyncCB),
```

Comment:
Please avoid abbreviations. `asyncCallback` is easier to read and with auto-complete no more work to type :)